easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3_amend_search_for_nvml_lib.patch

```diff
@@ -8,7 +8,7 @@ diff -ru gromacs-2016.orig/cmake/FindNVML.cmake gromacs-2016/cmake/FindNVML.cmak
    # passed to the installer, or the root where they later found the
```

Comment:
Please include an explanation at the top as to why you've changed the original patch